posthog/hogql_queries/web_analytics/web_analytics_query_runner.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+
+from posthog.caching.insights_api import BASE_MINIMUM_INSIGHT_REFRESH_INTERVAL
+from posthog.hogql_queries.query_runner import QueryRunner
+
+
+class WebAnalyticsQueryRunner(QueryRunner, ABC):
+    def _is_stale(self, cached_result_package):
+        return True
+
+    def _refresh_frequency(self):
+        return BASE_MINIMUM_INSIGHT_REFRESH_INTERVAL
```

Comment:
Out of scope / random thought: if we're going to add many queries, these should also get defaults in the base runner, and be overidden on a as-needed basis.